src/pybind/mgr/dashboard/frontend/src/app/shared/constants/app.constants.ts

```diff
@@ -0,0 +1,7 @@
+export class AppConstants {
+  public static readonly organization = 'ceph';
+  public static readonly projectName = 'Ceph Manager Dashboard';
+  public static readonly copyRight =
+    'Copyright(c) ' + new Date().getFullYear() + ' Ceph contributors.';
```

Comment:
The copyright year establishes [since when the authorship of something is claimed](https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated). By updating this value to the client's date, we are rendering this copyright warning useless. Let's go back to the initial version, or at least a [build-time value](https://medium.com/@kaustubhtalathi/inject-build-timestamp-into-angular-cli-build-output-files-to-verify-deployments-49416e2e13b3).

Reply:
ack, going with build-time value looks more appropriate